### PR TITLE
Update the directory structure after like 7 months

### PR DESCRIPTION
**Pull request by akionsight**
it had been severely outdated so just updated it

### CODE REVIEW

**Comment by akionsight** [comment]
O ok checks passed

File: STRUCTURE.md [hunk 1]
```diff
@@ -1,154 +1,357 @@
...
+│   │   │   ├── judge_vote.html
+│   │   │   ├── submissions_list.html
+│   │   │   ├── submit.html
+│   │   │   └── view_team.html
+│   │   └── weekly
+│   │       └── index.html
+│   ├── urls.py
+│   ├── views.py
+│   └── wsgi.py
+└── twtcodejam.net
```
Comment (takos22) [inline STRUCTURE.md:178]: You need to be inside this dir when doing the tree command

File: STRUCTURE.md [hunk 2]
```diff
@@ -1,154 +1,357 @@
...
+    ├── SETUP.md
+    ├── STRUCTURE.md
+    ├── TWT
+    │   ├── __init__.py
+    │   ├── apps
+    │   │   ├── challenges
+    │   │   │   ├── __init__.py
+    │   │   │   ├── admin.py
+    │   │   │   ├── migrations
+    │   │   │   │   ├── 0001_initial.py
```
Comment (takos22) [inline STRUCTURE.md:191]: Remove these files and replace them with `...`

File: STRUCTURE.md [hunk 3]
```diff
@@ -1,154 +1,357 @@
...
+    │   │   │       ├── stop_team.py
+    │   │   │       ├── stop_voting.py
+    │   │   │       ├── test.py
+    │   │   │       ├── unreleased.py
+    │   │   │       └── view.py
+    │   │   ├── timathon
+    │   │   │   ├── __init__.py
+    │   │   │   ├── admin.py
+    │   │   │   ├── migrations
+    │   │   │   │   ├── 0001_initial.py
```
Comment (takos22) [inline STRUCTURE.md:235]: Remove these files and replace them with `...`

File: STRUCTURE.md [hunk 4]
```diff
@@ -1,154 +1,357 @@
...
+    │   │   │   ├── bootstrap.min.css
+    │   │   │   ├── index.css
+    │   │   │   ├── judge_home.css
+    │   │   │   ├── logout.css
+    │   │   │   ├── new.css
+    │   │   │   ├── submissions_list.css
+    │   │   │   ├── timathonHome.css
+    │   │   │   └── timathonTeam.css
+    │   │   ├── fonts
+    │   │   │   ├── Lovelo-Black.woff
```
Comment (takos22) [inline STRUCTURE.md:299]: Remove these files and replace them with `...`

**Review by takos22 — CHANGES_REQUESTED** [review]
You need to cd in the right dir and replace the useless files (migrations and fonts) with `...`

**Comment by mohamed040406** [comment]
why do we even need a directory tree?

**Comment by akionsight** [comment]
> 
> 
> why do we even need a directory tree?

Yeah you know what, I thought about it for a little while. and I think we do not need one
I think I should close the PR and the Directory Structure should be removed

**Pull request closed without merging**